lms/static/lms/js/spec/main.js

```diff
@@ -1,4 +1,4 @@
-(function(requirejs, define) {
+(function(requirejs) {
```

Comment:
Reviewers: I moved these files beneath the LMS namespace (and did the same for the CMS ones) now that I've introduced a spec directory there. IMO all LMS-specific files should live beneath the LMS namespace directory.